### PR TITLE
feat: adding the footer

### PR DESCRIPTION
**Pull request by Arzoid29**

### CODE REVIEW

File: src/model/api/routes-data/data.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    whySalesAssist = 'Why SalesAssist',
+    pricing = 'Pricing',
+    about = 'About Us',
+    qAndA = 'Q&A',
+    howItWorks = 'How it Works',
+    Benefits = 'Benefits',
+    actions = 'Actions',
+    bookADemo = 'Book Demo',
+    legal = 'Legal',
+    policy = 'Privacy Policy',
```
Comment (brya0x) [inline src/model/api/routes-data/data.ts]: **Change**: use translates

**Review by code-bryan — CHANGES_REQUESTED** [review]

File: src/components/organism/sections/footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                  height={224}
+                />
+              ) : (
+                <Image
+                  src="/images/logo.png"
+                  alt={""}
+                  width={500}
+                  height={224}
+                />
+              )}
```
Comment (brya0x) [inline src/components/organism/sections/footer/index.tsx:54]: **Change**: you can refactor this to a logo component that will render depending on the kind

File: src/components/organism/sections/footer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            </div>
+            <h4 className="hidden md:block text-[0.75rem]">
+              © Shiplove Inc. {new Date().getFullYear()} All Rights Reserved
```
Comment (brya0x) [inline src/components/organism/sections/footer/index.tsx:57]: **Change**: use translates here

File: src/components/organism/sections/footer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                  key={i}
+                  href={s.url}
+                  target="_blank"
+                  rel="noreferrer"
+                  className={socialClasses}
+                >
+                  {s.icon}
+                </a>
+              ))}
+            </div>
```
Comment (brya0x) [inline src/components/organism/sections/footer/index.tsx:102]: **Change**: this section can be refactor

File: src/components/organism/sections/footer/index.tsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+                <p className="px-0 hover:text-orange-500">{link.name}</p>
+              </NextLink>
+            ))}
+          </div>
+
+          {/* social media */}
+          <div className="md:w-1/2 h-full flex justify-end gap-2 md:gap-6 flex-col md:flex-row w-full md:pb-12">
+            <h4 className="block md:hidden text-[0.75rem] text-gray-400">
+              © Shiplove Inc. {new Date().getFullYear()} All Rights Reserved
+            </h4>
```
Comment (brya0x) [inline src/components/organism/sections/footer/index.tsx:89]: **Change**: i saw this twice created a component for it

File: src/model/api/routes-data/data.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        { path: '/about-us', name: RouteNames.about },
+        { path: calendlyLink, name: RouteNames.bookADemo },
+    ]
+}
+
+const contact = {
+    title: 'Contact Us',
+    links: [
+        { path: 'mailto:help@salesassist.io', name: 'help@salesassist.io' },
+        { path: '#', name: '251 Little Falls Drive, Wilmington, Delaware 19808' },
```
Comment (brya0x) [inline src/model/api/routes-data/data.ts:51]: **Change**: the title too need translates

**Review by code-bryan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**